src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,16 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
```

Comment:
Consider using default exports instead of named exports for consistency and better readability. 🤔